libraries/tools/kotlin-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/internal/kapt/Kapt3KotlinGradleSubplugin.kt

```diff
@@ -447,7 +447,13 @@ class Kapt3GradleSubplugin @Inject internal constructor(private val registry: To
                 }
             }
 
-            kaptTask.kaptClasspathConfigurations = kaptClasspathConfigurations
+            val kaptClasspathConfiguration =
+                project.configurations.create("_kaptClasspath" + kaptTask.name).setExtendsFrom(kaptClasspathConfigurations).also {
```

Comment:
nit: add "_" before kaptTask.name (looks like it starts with lowercase?)

Reply:
done